_data/nav.yml

```diff
@@ -85,8 +85,10 @@ How to use USWDS:
   - href: /documentation/accessibility/
   - href: /performance/
   - href: /documentation/code-guidelines/
-  - text: Migrating to USWDS 2
+  - text: Migrating to USWDS 3
```

Comment:
@bonnieAcameron @mahoneycm 
Normally we match the nav text with the page title. 

The title of the page is "USWDS 3.0 Migration Guide" and the nav text is "Migrating to USWDS 3". Can we make these items match? 

Also just quickly clicking between migration guides for the two versions, there is a slight inconsistency in naming convention, where we call it USWDS 2 and USWDS 3.0 (vs USWDS 3). Just wanted to make sure this was intentional!

Reply:
Let's do 3.0 instead! Thanks for catching.

Reply:
Happy to change this to whatever we deem appropriate!

Reply:
Resolved in https://github.com/uswds/uswds-site/pull/1529/commits/7942bc2cbd15345bd6495de46a3f910efa57a1e9 !